include/SDL3/SDL_main.h

```diff
@@ -114,7 +117,7 @@
  *
  * \since This macro is available since SDL 3.2.0.
  */
-#define SDL_MAIN_AVAILABLE
+#define SDL_MAIN_AVAILABLE 1
```

Comment:
FYI, I changed these because I think the wiki does not currently pick up defines that don't have any values. For example, https://wiki.libsdl.org/SDL3/SDL_MAIN_AVAILABLE doesn't exist.

Reply:
That's definitely a bug in wikiheaders, I'll take a look at that.

Reply:
That bug is fixed in a0fa64a91ca8ab623cb5eb6be6331d134b8f45c3.